### ISSUE TITLE
Failed boot of grub on UEFI

### ISSUE THREAD

**Issue by gkisuze** [HankB/Linux_ZFS_Root#15]
When tested with dell optiplex 7040, PC becomes unbootable in both UEFI and legacy mode. Problem arises from the way grub handles UEFI boot process.
There is a fix to the problem when refind is installed to replace grub. I wish to replace grub with refind if UEFI mode is selected. Any ideas?
#apt install refind
#refind-install
#refind-install --usedefault /dev/sda2 
#reboot

**Comment by HankB** [comment]
Is it possible to install refind after the install is complete?

**Comment by gkisuze** [comment]
Yes it is. Just before reboot. Use the three commands I indicated. i.e
#apt install refind
#refind-install
#refind-install --usedefault /dev/sda2
It works out well.
